zk-token-sdk/src/zk_token_proof_instruction.rs

```diff
@@ -258,10 +248,8 @@ pub enum ProofInstruction {
 
     /// Verify a ciphertext-commitment equality proof.
     ///
-    /// This instruction can be configured to optionally initialize a proof context state account.
-    /// If creating a context state account, an account must be pre-allocated to the exact size of
-    /// `ProofContextState<CiphertextCommitmentEqualityProofContext>` and assigned to the ZkToken
-    /// proof program prior to the execution of this instruction.
+    /// A ciphertext-commitment equality proof certifies that an ElGamal ciphertext and a Pedersen
+    /// commitment encrypts/encodes the same message.
```

Comment:
```suggestion
    /// A ciphertext-commitment equality proof certifies that an ElGamal ciphertext and a Pedersen
    /// commitment encrypt/encode the same message.
```
But do we need both terms `encrypt` and `encode`?